Scripts/Python/xHighLevelStarTrekDoor.py

```diff
@@ -94,46 +99,29 @@ def OnServerInitComplete(self):
                 PtDebugPrint("ERROR: xHighLevelStarTrekDoor.OnServerInitComplete():\tERROR: age sdl read failed, defaulting door enabled value")
         else:
             self.DoorEnabled = 1
-        
+
         try:
-            self.DoorState = self.SDL['DoorState'][0]
+            self.SDL['DoorState'][0]
         except:
             self.SDL['DoorState'] = (0,)
-            self.DoorState = self.SDL['DoorState'][0]
-         
-        PtDebugPrint("xHighLevelStarTrekDoor: self.SDL = %d" % self.DoorState)
+
+        PtDebugPrint(f"xHighLevelStarTrekDoor: {self.SDL['DoorState'][0]=}")
 
         if not PtIsSolo():
-            
+
             PtDebugPrint("xHighLevelStarTrekDoor: Somebody is already in the age. Attempting to sync states.")
 
-            if self.DoorState == doorSDLstates['opening'] or self.DoorState == doorSDLstates['movingopen'] or self.DoorState == doorSDLstates['opentoclose']:
-                respOpenDoor.run(self.key,netPropagate=0)
-                self.DoorStack.append("respOpenDoor.run(self.key,netPropagate=0)")
-
-                '''
-                respOpenDoor.run(self.key,fastforward=1)
-                self.DoorState = doorSDLstates['open']
-                PtDebugPrint("xHighLevelStarTrekDoor: Door is open.")
-                '''
-            
-            elif self.DoorState == doorSDLstates['closing'] or self.DoorState == doorSDLstates['movingclosed'] or self.DoorState == doorSDLstates['closetoopen']:
-                respCloseDoor.run(self.key,netPropagate=0)
-                self.DoorStack.append("respCloseDoor.run(self.key,netPropagate=0)")
-                '''
-                respCloseDoor.run(self.key,fastforward=1)
-                self.DoorState = doorSDLstates['closed']
-                PtDebugPrint("xHighLevelStarTrekDoor: Door is closed.")
-                '''
-            
-            elif self.DoorState == doorSDLstates['open']:
-                respOpenDoor.run(self.key,fastforward=1)
-                
+            if self.SDL['DoorState'][0] in (doorSDLstates['opening'], doorSDLstates['movingopen'], doorSDLstates['opentoclose']):
```

Comment:
```suggestion
            if self.SDL['DoorState'][0] in (doorSDLstates.opening, doorSDLstates.movingopen, doorSDLstates.opentoclose):
```

`doorSDLstates` should now be using members thusly. You'll need to apply these changes throughout the file.